Tox and Docs

Running tox after generating the docs results in the following.

Probably we can tell tox to ignore the doc build artifacts.

```
tox -e check
check: install_deps> python -I -m pip install black check-manifest flake8 flake8-bugbear flake8-colors flake8-print isort twine
check: commands[0]> flake8 .
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:91:1: T201 print found.
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:96:1: T201 print found.
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:98:1: T201 print found.
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:103:1: T201 print found.
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:115:1: E402 module level import not at top of file
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:117:1: T201 print found.
./docs/build/html/_downloads/7ae0b9a41836bdfffb6e66391d0a0718/configuration.py:121:1: T201 print found.
./docs/source/auto_tutorials/configuration.py:91:1: T201 print found.
./docs/source/auto_tutorials/configuration.py:96:1: T201 print found.
./docs/source/auto_tutorials/configuration.py:98:1: T201 print found.
./docs/source/auto_tutorials/configuration.py:103:1: T201 print found.
./docs/source/auto_tutorials/configuration.py:115:1: E402 module level import not at top of file
./docs/source/auto_tutorials/configuration.py:117:1: T201 print found.
./docs/source/auto_tutorials/configuration.py:121:1: T201 print found.
check: exit 1 (2.31 seconds) /Users/gbwright/work/ASPIRE-Python.conda_opts> flake8 . pid=54341
  check: FAIL code 1 (9.76=setup[7.45]+cmd[2.31] seconds)
  evaluation failed :( (10.14 seconds)
  ```

--- a/tox.ini
+++ b/tox.ini
@@ -69,6 +69,9 @@ extend-ignore = E203, E501
 per-file-ignores =
   __init__.py: F401
  gallery/tutorials/configuration.py: T201, E402
+ # Ignore Sphinx gallery builds
+ docs/build/html/_downloads/*/*.py: T201, E402
+ docs/source/auto*/*.py: T201, E402
 
 [isort]
 default_section = THIRDPARTY

